topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -121,6 +107,21 @@ def populate_item(self, metadata_row: Dict[str, str], asset: Optional[Asset] = N
         item.add_extension(StacExtensions.linz.value)
         item.add_extension(StacExtensions.version.value)
 
+    def population_collection(self, collection: Collection) -> None:
```

Comment:
As your function change the state of the `collection`, it's more clear to use a verb (i.e. `populate` instead of `population`) for the function name.